dev-java/javacup/javacup-0.11b_p20151001-r1.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
+
+MY_PV=${PV/_beta/-}
+MY_PV=${MY_PV/_p/-}
+MY_PV=${MY_PV#0.}
+MY_P=java-cup-${MY_PV%-*}
+
+DESCRIPTION="CUP Parser Generator for Java"
+HOMEPAGE="http://www2.cs.tum.edu/projects/cup/"
+SRC_URI="http://www2.cs.tum.edu/projects/cup/releases/java-cup-src-${MY_PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+DEPEND="dev-java/ant-core:0
+	dev-java/jflex:0
+	>=virtual/jdk-1.8:*"
+
+RDEPEND=">=virtual/jre-1.7"
```

Comment:
though really not an issue, this should go to 1.8 aswell

Reply:
Oops, adjusted.